modules/oadp-backup-restore-cr-issues.adoc

```diff
@@ -75,13 +75,78 @@ time="2023-02-17T16:33:13Z" level=error msg="Error backing up item" backup=opens
 $ oc delete backup <backup> -n openshift-adp
 ----
 
-. If required, clean up the stored data on the `BackupStorageLocation` to free up space.
+. If required, free up space by cleaning up the stored data on the `BackupStorageLocation`.
 
-. Apply label `velero.io/csi-volumesnapshot-class=true` to the `VolumeSnapshotClass` object:
+. Apply the label `velero.io/csi-volumesnapshot-class=true` to the `VolumeSnapshotClass` object:
 +
 [source,terminal]
 ----
 $ oc label volumesnapshotclass/<snapclass_name> velero.io/csi-volumesnapshot-class=true
 ----
 
-. Create a new `Backup` CR.
\ No newline at end of file
+. Create a new `Backup` CR.
+
+[id="datamover-backup-cleanup_{context}"]
+== Datamover backup cleanup
+
+[NOTE]
+====
+Applies to OADP 1.1.x and newer.
+====
+
+Datamover may leave behind some older snapshots in the bucket, causing the system to recover an older backup. To prevent recovering an outdated backup, delete the folder with old snapshot before creating a new snapshot.
+
+.Remove snapshots in bucket
+The snapshots are saved in your bucket specified in the `/<protected-ns> folder`, DPA'.spec.backupLocation.objectStorage.bucketw. Delete this folder to delete *all* snapshots in the bucket.
+
+Additional sub-folder(s) with the prefix z/<volumeSnapshotContent name>-pvc` may be inside the `/<protected-ns>` folder. These folders hold the _volumeSnapshotContent_ backups that datamover created per PVC. Delete each of these sub-folder to delete *a single* snapshot in your bucket.
+
+.Remove cluster resources
+The following scenarios can remove the cluster resources:
+
+. Datamover completes
+
+volumeSnapshotBackup/VolumeSnapshotRestore CRs are still in the application namespace.
+
+* Datamover backup:
+
++
+[source, text]
+----
+oc delete vsb -n <app-ns> --all
```

Comment:
We are suggesting to cleanup all the things.

oc delete volumesnapshot -A --all
oc delete volumesnapshotcontent --all
oc delete pvc -n <protected-ns> --all
oc delete replicationsource -n <protected-ns> --all

Reply:
Added in lines 117-120